migrator/migrations/m_96_to_m_97_modify_default_vulnreportcreator_role/migration.go

```diff
@@ -0,0 +1,94 @@
+package m96tom97
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/rocksdbmigration"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	permissionsUtils "github.com/stackrox/rox/pkg/auth/permissions/utils"
+	"github.com/stackrox/rox/pkg/auth/role"
```

Comment:
Let's not rely on vars from different package here in the migrator code and call them out here like you do for bucket names, similar to what, e.g., 90 to 91 migration does. Those vars can change and might render the migration incorrect.

As a bonus, this will also significantly reduce the size of the PR because you won't need to move the file : ).

Reply:
Vars you mean the permission sets and role names? I can't completely get rid of that dependency since i need the name. Also I do believe that pkg is the right place for those vars/constants, no?

Reply:
This is the safest way to get the name in a migration:
```
var (
	imageResource             = "Image"
	vulnMgmtRequestsResource  = "VulnerabilityManagementRequests"
	vulnMgmtApprovalsResource = "VulnerabilityManagementApprovals"
...
```
Imagine the value of the referenced var changes in the future, then the migration will simply become incorrect. After that, you can get rid of the dependency.

> Also I do believe that pkg is the right place for those vars/constants, no?

This is a different conversation and hence a different PR : ). I wonder if you will still want it after the task at hand is solved 😛.